### DISCUSSION TITLE
Unable to verify - Feature your project in the WalletConnect explorer

### DISCUSSION THREAD

**Discussion by brklyn8900** [WalletConnect/walletconnect-monorepo#3583] [Developer Support]
While submitting my dapp to be part of the WalletConnect marketplace I am asked to verify by signing a transaction using https://lab.web3modal.com/. The dapp is on the Koinos network, I'm unable to sign the transaction. Koinos is not available in the select network modal. 

"Wallet Testing - Could not verify if testing has been done on this wallet. As we do not have available V2 metric data for Sign. Please test your integration with our test dapp and submit for approval if successful. Link: https://lab.web3modal.com/"

**Comment by glitch-txs** [comment]
hey it seems like you've submited your project as a Wallet instead of a Dapp?

**Comment by brklyn8900** [comment]
I submitted it as a wallet because it is a wallet.

**Reply by glitch-txs** [reply]
ahh okay thanks, have you been able to test it on a custom dapp?

**Comment by brklyn8900** [comment]
I have, and it works. Is it possible the problem is that the wallet is on the Koinos network which is relatively new? But there is another wallet on there that has been approved (https://walletconnect.com/explorer/konio) and I'm not sure how. 

I'm trying to verify here: 
<img width="1840" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/70003/f4f71b37-7d0a-4f34-9bc0-60c4198c6cfb">

But I keep getting the error: "Connection request declined"

**Comment by roaminro** [comment]
Hi, @glitch-txs ! 

Our application for the cloud is still being rejected with the message below even though our wallet CANNOT be tested with https://lab.web3modal.com/ because our wallet is for a blockchain called Koinos which is NOT EVM compatible.

Can you tell the team who's in charge of reviewing the application to read the testing instructions provided in the application form as they explicitly mention this wallet cannot be tested with https://lab.web3modal.com (because it's an EVM only webiste) but has to be tested with https://armana-group.github.io/walletconnect-koinos-sdk-js/ instead.

Here's my discord handle if more private details are needed to move forward with this issue: `@_roamin_`

Thanks in advance for your help!

The rejection message we keep getting: 
"Wallet Testing - Could not verify if testing has been done on this wallet. As we do not have available V2 metric data for Sign. Please test your integration with our test dapp and submit for approval if successful. Link: https://lab.web3modal.com/"

**Reply by glitch-txs** [reply]
hi @roaminro @brklyn8900 could you share your project id?

**Reply by roaminro** [reply]
Hi, @glitch-txs! Thanks for you answer, here's the project id: `8c07baf0d24ccc95937f638275811c1f`

**Reply by glitch-txs** [reply]
Thanks, should be approved now 🫡

**Comment by brklyn8900** [comment]
Bump
